tests/robustness/crash_consistency_test/crash_consistency_test.go

```diff
@@ -0,0 +1,262 @@
+//go:build darwin || (linux && amd64)
+// +build darwin linux,amd64
+
+package consistency
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/kopia/kopia/tests/recovery/blobmanipulator"
+	"github.com/kopia/kopia/tests/tools/kopiarunner"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConsistencyWhenKill9AfterModify(t *testing.T) {
+	// create, connect repository
+	dataRepoPath := path.Join(*repoPathPrefix, dirPath+dataPath)
+	baseDir := makeBaseDir()
+	bm, err := blobmanipulator.NewBlobManipulator(baseDir, dataRepoPath)
+	if err != nil {
+		if errors.Is(err, kopiarunner.ErrExeVariableNotSet) {
+			log.Println("Skipping recovery tests because KOPIA_EXE is not set")
```

Comment:
You want to mark this test as skipped instead of success, so better to use the testing skip interface for that.

```suggestion
			t.Skip("Skipping recovery tests because KOPIA_EXE is not set")
```